Functions.Templates/Bindings/bindings.json

```diff
@@ -115,6 +115,16 @@
         }
       ]
     },
+    {
+      "type": "rabbitMQTrigger",
+      "displayName": "$rabbitMQTrigger_displayName",
+      "direction": "trigger",
+      "enabledInTryMode": true,
+      "documentation": "$content=Documentation\\rabbitMQTrigger.md",
+      "extension": {
+        "id": "Microsoft.Azure.WebJobs.Extensions.RabbitMQ", "version": "1.0.0-beta"
+      }

```

Comment:
@yojagad @pragnagopa @soninaren this is currently breaking VS Code tests on the staging feed because it doesn't have a "settings" array, like the other bindings. Can we get that added before it moves from staging to prod? I can fix our extension to more gracefully handle a binding with no settings going forward, but would certainly be easier to fix here first